release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1179,7 +1179,11 @@ This update corrects the accepted label specification so that the plug-in import
 [id="ocp-4-10-installer-bug-fixes"]
 ==== Installer
 
-Previously, pre-flight checks did not account for {rh-openstack-first} resource utilization. As a result, those checks failed with an incorrect error message when utilization, rather than quota, impeded installation. Pre-flight checks now process both {rh-openstack} quota and utilization. The checks fail with correct error messages if the quota is sufficient but resources are not. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001317[*BZ#2001317*])
+* Previously, pre-flight checks did not account for {rh-openstack-first} resource utilization. As a result, those checks failed with an incorrect error message when utilization, rather than quota, impeded installation. Pre-flight checks now process both {rh-openstack} quota and utilization. The checks fail with correct error messages if the quota is sufficient but resources are not. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001317[*BZ#2001317*])
+
+* Before this update, the oVirt Driver could specify ReadOnlyMany (ROX) and ReadWriteMany (RWX) access modes when creating a PVC from a configuration config file. This caused an error as the driver does not support shared disks, and so cannot support these access modes. With this update, the access mode has been limited to single node access. The system prevents any attempt to specify ROX or RWX when creating PVC and logs an error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1882983[*BZ#1882983*])
+
+* Previously, disk uploads in the Terraform provider were not handled properly. As a result, the OpenShift installer failed. With this update, disk upload handling has been fixed, and disk uploads succeed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917893[*BZ#1917893*])
```

Comment:
I think this should be the OpenShift installation program per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#_i).

Reply:
{product-title} installation program, but yes.